apps/builder/app/builder/features/props-panel/controls/url.tsx

```diff
@@ -211,9 +222,94 @@ const BaseAttachment = ({ prop, onChange, onSoftDelete }: BaseControlProps) => (
   </Row>
 );
 
+// store that contains IDs off all instances in the selected page
+const pageInstancesStore = computed(
```

Comment:
this could be useful globally I guess, no? I have a feeling we must be potentially even doing this already somewhere @TrySound

Reply:
Yeah, wasn't sure about this too. @TrySound What do you think?

Reply:
See findTreeInstanceIds in project-build